src/Api/AdminConsole/Models/Response/Organizations/OrganizationAutoEnrollStatusResponseModel.cs

```diff
@@ -4,12 +4,14 @@ namespace Bit.Api.AdminConsole.Models.Response.Organizations;
 
 public class OrganizationAutoEnrollStatusResponseModel : ResponseModel
 {
-    public OrganizationAutoEnrollStatusResponseModel(Guid orgId, bool resetPasswordEnabled) : base("organizationAutoEnrollStatus")
+    public OrganizationAutoEnrollStatusResponseModel(Guid orgId, bool resetPasswordEnabled, String organizationName) : base("organizationAutoEnrollStatus")
```

Comment:
:art: I think we usually opt for `string` over `String`

Reply:
Nice, fixed in [819dfc8](https://github.com/bitwarden/server/pull/5627/commits/819dfc831f389014955dc1af095fbb3cac8d8f32)